src/components/tables/cells/fieldSelection/FieldActions.tsx

```diff
@@ -73,21 +77,48 @@ function FieldActions({ bindingUUID, field, constraint }: Props) {
                 exclusive
                 sx={outlinedToggleButtonGroupStyling}
             >
-                <FieldActionButton
-                    messageId="fieldSelection.table.cta.includeField"
-                    selectedValue={selection?.mode ?? null}
+                <OutlinedToggleButton
+                    color="success"
+                    disabled={formActive || !recommendFields[bindingUUID]}
+                    onClick={() => {
+                        const singleValue =
+                            selection?.mode !== 'default' ? 'default' : null;
+
+                        const selectionType = evaluateSelectionType(
+                            recommendFields[bindingUUID],
+                            'default',
+                            selection?.mode ?? null,
+                            singleValue
+                        );
+
+                        setSingleSelection(
+                            bindingUUID,
+                            field,
+                            selectionType,
+                            selection?.meta
+                        );
+                    }}
+                    selected={selection?.mode === 'default'}
+                    size="small"
+                    value="default"
+                >
+                    {intl.formatMessage({
+                        id: 'fieldSelection.table.cta.selectField',
+                    })}
+                </OutlinedToggleButton>
+
+                <OutlinedToggleButton
+                    color="warning"
+                    selected={selection?.mode === 'require'}
```

Comment:
Can we use the `value' prop on the group so we don't have to manage this ourselves?